homeassistant/helpers/entity_component.py

```diff
@@ -19,11 +19,13 @@
 from homeassistant.util.async import (
     run_callback_threadsafe, run_coroutine_threadsafe)
 import homeassistant.util.dt as dt_util
+from .entity_registry import EntityRegistry
```

Comment:
This is a relative import.

Reply:
Sorry, I meant I don't know if I like them more

Reply:
Ok, then I'll employ a lax attitude until someone says otherwise, if it shows up in another PR. :tropical_drink: